src/concrete/ml/quantization/base_quantized_op.py

```diff
@@ -559,7 +560,10 @@ def _prepare_quantized_input(self, input_: QuantizedArray) -> QuantizedArray:
         # but when parsing the ONNX graph, some options can be overwritten. Thus
         # when evaluating QAT layers we ignore one of these options to allow the
         # override.
-        if quant_opts.is_equal(input_.quantizer.quant_options, ignore_sign_qat=True):
+        if (
+            quant_opts.is_equal(input_.quantizer.quant_options, ignore_sign_qat=True)
+            or input_.quantizer.quant_options.is_precomputed_qat
```

Comment:
before we were overriding the op input quantization mainly for the model input. now we override every time there is a BrevitasQuant layer before the op

Reply:
Thanks for the explanation, maybe you should update the comment accordingly.